src/widgets/archivelistwidgetitem.cpp

```diff
@@ -1,24 +1,16 @@
 #include "archivelistwidgetitem.h"
+
 #include "utils.h"
 
 #define FIELD_WIDTH 6
 
 ArchiveListWidgetItem::ArchiveListWidgetItem(ArchivePtr archive)
     : _widget(new QWidget)
 {
-    _widget->installEventFilter(this);
     _ui.setupUi(_widget);
-
-    _ui.inspectButton->setToolTip(_ui.inspectButton->toolTip()
-                                   .arg(_ui.actionInspect->shortcut()
-                                        .toString(QKeySequence::NativeText)));
-    _ui.restoreButton->setToolTip(_ui.restoreButton->toolTip()
-                                   .arg(_ui.actionRestore->shortcut()
-                                        .toString(QKeySequence::NativeText)));
-    _ui.deleteButton->setToolTip(_ui.deleteButton->toolTip()
-                                   .arg(_ui.actionDelete->shortcut()
-                                        .toString(QKeySequence::NativeText)));
-
```

Comment:
Why did you move these pieces around? The global style in the constructors should be:
1. UI initializations;
2. Setup/logic code;
3. Signals and slots;